sdk/python/tests/compiler/testdata/artifact_location.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp
+from kfp import dsl
+from kubernetes.client import V1SecretKeySelector
+
+
+@dsl.pipeline(
+    name="custom_artifact_location_pipeline",
+    description="""A pipeline to demonstrate how to configure the artifact
+    location for all the ops in the pipeline.""",
```

Comment:
maybe we can add another comment here that this assumes the pipeline will be deployed/run in the `kubeflow` namespace?